.github/ISSUE_TEMPLATE/1-feature_requests.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Requests
+about: Suggest an enhancement or new feature for the Grafana project
+labels: 'type/feature-request'
+---
+
+<!-- Please only use this template for submitting feature requests -->
+
+**What would you like to be added**:
+
+[Add a brief description of what the feature or enhancement should do.]
+
+**Why is this needed**:
```

Comment:
I'd maybe swap the order of these two: 
1. Problem
2. Solution
3. Consequences/Impact

WDYT? Similar to a Product DNA

Reply:
Sounds good to me. I'll change it